lighthouse-core/test/audits/screenshot-thumbnails-test.js

```diff
@@ -30,14 +30,14 @@ describe('Screenshot thumbnails', () => {
         const framePath = path.join(__dirname,
             `../fixtures/traces/screenshots/progressive-app-frame-${index}.jpg`);
         const expectedData = fs.readFileSync(framePath, 'base64');
-        assert.equal(expectedData.length, result.data.length);
+        expect(result.data.length - 'data:image/jpeg;base64,'.length).toEqual(expectedData.length);
```

Comment:
weird test :)

Reply:
heh, I can slice it instead to make it a little more explicit :)

Reply:
> > weird test :)
>
> heh, I can slice it instead to make it a little more explicit :)

ha, I meant the original test is a weird one, but I like the change :)